packages/react/src/components/Forms/Select/mono-common.tsx

```diff
@@ -134,45 +134,47 @@ export const SelectMonoAux = ({
             >
               <div className="c__select__inner">
                 <div className="c__select__inner__value">{children}</div>
-                <div className="c__select__inner__actions">
-                  {clearable && !disabled && downshiftReturn.selectedItem && (
-                    <>
-                      <Button
-                        color="tertiary-text"
-                        size="nano"
-                        aria-label={t(
-                          "components.forms.select.clear_button_aria_label",
-                        )}
-                        className="c__select__inner__actions__clear"
-                        onClick={(e) => {
-                          downshiftReturn.selectItem(null);
-                          e.stopPropagation();
-                        }}
-                        icon={<span className="material-icons">close</span>}
-                        type="button"
-                      />
-                      <div className="c__select__inner__actions__separator" />
-                    </>
-                  )}
+                {!props.isLoading && (
```

Comment:
The bug you are describing in your comment https://github.com/openfun/cunningham/pull/305#issuecomment-2195785346 is due to this change.

I think as you are mounting/unmounting this component according to the loading state, you are breaking some refs used by downshift under the hood.